lib/lita/handlers/dependency_updater.rb

```diff
@@ -38,9 +38,17 @@ class DependencyUpdater < Handler
         chefdk: {
           pipeline: "chefdk-trigger-release",
           github_url: "git@github.com:chef/chef-dk.git",
-          version_bump_command: "bundle install && bundle exec rake version:bump",
+          version_bump_command: "bundle install && bundle exec rake version:bump && git checkout .bundle/config",
```

Comment:
You can probably remove this, I think I just copy-pasta'd from the other one and forgot to take it out. Alternatively, could merge everything into one project config file.